### ISSUE TITLE
Download speed 2MB/s

### ISSUE THREAD

**Issue by 3941** [Nexus-Mods/Nexus-Mod-Manager#1056]
That's what the Nexus has been offering for free for a while now.
Shouldn't that be reflected in NMM?

**Comment by 3941** [comment]
Actually I think it might be loading faster than 1024kb/s... it just doesn't say so in the lower left.

**Comment by DuskDweller** [comment]
Yes it's only a visual issue, we'll default it to 2048 in the next release since there's no way for us to know whether a download is being server at 1MB for adblocking or not.
